components/expenses/ExpensePayeeDetails.js

```diff
@@ -186,7 +186,10 @@ const ExpensePayeeDetails = ({ expense, host, isLoading, borderless, isLoadingLo
             />
           </Box>
           <div data-cy="expense-summary-payout-method-data">
-            <PayoutMethodData payoutMethod={expense.payoutMethod} isLoading={isLoadingLoggedInUser} />
+            <PayoutMethodData
+              payoutMethod={expense.draft ? expense.draft?.payoutMethod : expense.payoutMethod}
```

Comment:
We can simplify this condition, it's double-checking the existence of `expense.draft`

```suggestion
              payoutMethod={expense.draft?.payoutMethod ?? expense.payoutMethod}
```

Reply:
Done. 👍🏽